data-imports/utils.py

```diff
@@ -152,7 +152,7 @@ def hpd_csv2sql(description, input_csv_url, sep_char,
             log.info("Processing chunk {}".format(chunk_num))
 
             chunk_pickle_file = pickle_file.format(chunk_num)
-            df = process_df(df, log, description, save_pickle, chunk_pickle_file,
+            df = process_df(df, log, description, save_pickle, pickle_file,
```

Comment:
This is supposed to be chunk_pickle_file - when importing the CSV in chunks (only used for 311 right now), the user provides a pickle file name format, which gets formatted with the number (into chunk_pickle_file). Passing pickle_file will cause each chunk's pickle to overwrite that of the previous chunk.

I should probably pull this stuff out to a separate method so it's less confusing.